plugins/node/opentelemetry-instrumentation-undici/README.md

```diff
@@ -0,0 +1,72 @@
+# OpenTelemetry Undici/fetch Instrumentation for Node.js
```

Comment:
it may be good to document which version of semconv is being emitted from this instrumentation, see #1907

Reply:
> I'd be in favor of doing this in a separate PR

Yes I also think that's fine. I added added this instrumentation to the list on #1778 :slightly_smiling_face:

Reply:
Ah, if we are using a [local version](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1951#discussion_r1512607701) of semantic conventions here I think it would be good to mention the version at least. The table of emitted attributes and their descriptions can still be a follow-up, but I don't think I see a version here so I don't want to lose track of that (the other packages I had in mind included the semantic-convention package in package.json so it was still referenced just needed a bit of docs, I don't think I see a version here but I could be missing it)

Reply:
done! :)